CodeEdit/Features/NavigatorArea/ProjectNavigator/OutlineView/ProjectNavigatorMenu.swift

```diff
@@ -141,10 +143,7 @@ final class ProjectNavigatorMenu: NSMenu {
                 secondaryItems.append(.hex())
             }
 
-            // FIXME: Update the quickLook condition
-            if type.conforms(to: .data) {
-                secondaryItems.append(.quickLook())
```

Comment:
According to [Apple's documentation](https://developer.apple.com/documentation/quicklookui/qlpreviewitem), all URLs can be passed as preview items.